DOCS/man/input.rst

```diff
@@ -3454,6 +3457,48 @@ Property list
     and not using raw mode, the underlying content will be given (e.g. strings will be
     printed directly, rather than quoted and JSON-escaped).
 
+``menu-data`` (RW)
+    This property stores the raw menu definition, see `Context Menu`_ section.
+
+    ``type``
+        Menu item type, can be: ``separator``, ``submenu``, or empty.
+
+    ``title``
+        Menu item title, required if type is not ``separator``.
+
+    ``cmd``
+        Command to execute when the menu item is clicked.
+
+    ``shortcut``
+        Menu item shortcut key, appears to the right of the menu item.
+        A shortcut key does not have to be functional, it's just a visual hint.
+
+    ``state``
+        Menu item state, can be: ``checked``, ``disabled``, ``hidden``, or empty.
+
+    ``submenu``
+        Submenu items, required if type is ``submenu``.
+
+    When querying the property with the client API using ``MPV_FORMAT_NODE``, or with
+    Lua ``mp.get_property_native``, this will return a mpv_node with the following
+    contents:
+
+    ::
+
+        MPV_FORMAT_NODE_ARRAY
+            MPV_FORMAT_NODE_MAP (menu item)
+                "type"           MPV_FORMAT_STRING
+                "title"          MPV_FORMAT_STRING
+                "cmd"            MPV_FORMAT_STRING
+                "shortcut"       MPV_FORMAT_STRING
+                "state"          MPV_FORMAT_NODE_ARRAY[MPV_FORMAT_STRING]
+                "submenu"        MPV_FORMAT_NODE_ARRAY[menu item]
+
+    Writing to this property with the client API using ``MPV_FORMAT_NODE`` or with
+    Lua ``mp.set_property_native`` will trigger an immediate update of the menu if
+    mpv video output is currently active, you may observe the ``current-vo``
+    property to check if video output is active.
```

Comment:
@kasper93 @na-na-hi updated the document to address the vo requirement for updating menu.

The test script in PR body is also updated.